UI/css/icons.css

```diff
@@ -0,0 +1,50 @@
+@font-face {
+    font-family: 'Material Icons';
+    font-style: normal;
+    font-weight: 400;
+    src: url(material-fonts/MaterialIcons-Regular.eot); /* For IE6-8 */
+    src: local('Material Icons'),
+        local('MaterialIcons-Regular'),
+        url(material-fonts/MaterialIcons-Regular.woff2) format('woff2'),
+        url(material-fonts/MaterialIcons-Regular.woff) format('woff'),
+        url(material-fonts/MaterialIcons-Regular.ttf) format('truetype');
+}
+  
+.material-icons {
+    font-family: 'Material Icons';
+    font-weight: normal;
+    font-style: normal;
+    font-size: 24px;  /* Preferred icon size */
+    display: inline-block;
+    line-height: 1;
+    text-transform: none;
+    letter-spacing: normal;
+    word-wrap: normal;
+    white-space: nowrap;
+    direction: ltr;
+
+    /* Support for all WebKit browsers. */
+    -webkit-font-smoothing: antialiased;
+    /* Support for Safari and Chrome. */
```

Comment:
add a space before this comment. Just to be consistent with the following lines

Reply:
will do